Braze-Demo/BrazeManager.swift

```diff
@@ -119,11 +120,27 @@ extension BrazeManager {
     }
   }
   
+  /// Loops through an array of saved custom attribute data saved from storage. In the loop, the value. Once the attributes are logged, they are cleared from storage.
+  ///
+  /// `key` represents the attribute key and `value` represents the attribute value.
+  func logPendingAttributesIfNecessary() {
+    let remoteStorage = RemoteStorage(storageType: .suite)
+    guard let pendingAttributes = remoteStorage.retrieve(forKey: .pendingAttributes) as? [[String: Any]] else { return }
+    
+    for attribute in pendingAttributes {
```

Comment:
optional, may be cool to `foreach` this, you could even add a first class method for setting several attributes (`[String: Any]`) at once and something like:

```
pendingAttributes.forEach(setCustomAttributesWithKey)
```

Otherwise, I'd say `attribute` should be renamed to `attributes`